kube/job.go

```diff
@@ -17,16 +17,21 @@ func NewJob(role *model.Role, settings *ExportSettings) (helm.Node, error) {
 	// Jobs must have a restart policy that isn't "always"
 	switch role.Run.FlightStage {
 	case model.FlightStageManual:
-		podTemplate.Get("spec").Get("restartPolicy").SetValue("Never")
+		podTemplate.Get("spec/restartPolicy").SetValue("Never")
 	case model.FlightStageFlight, model.FlightStagePreFlight, model.FlightStagePostFlight:
-		podTemplate.Get("spec").Get("restartPolicy").SetValue("OnFailure")
+		podTemplate.Get("spec/restartPolicy").SetValue("OnFailure")
 	default:
 		return nil, fmt.Errorf("Role %s has unexpected flight stage %s", role.Name, role.Run.FlightStage)
 	}
 
-	job := newTypeMeta("extensions/v1beta1", "Job")
-	job.AddNode("metadata", helm.NewMapping("name", role.Name))
-	job.AddNode("spec", helm.NewNodeMapping("template", podTemplate))
+	kind := "extensions/v1beta1"
```

Comment:
`kind` should be `api` or `apiVersion`

Reply:
https://github.com/SUSE/fissile/pull/282/commits/13b9e729e2083fd62c79884643da80aba2183073